### PR TITLE
Update package wheel

### PR DESCRIPTION
**Pull request by Vyacheslav-Smirnov**
- Update long description

### CODE REVIEW

File: conda-recipe/meta.yaml [hunk 1]
```diff
@@ -13,6 +13,8 @@ requirements:
       - cmake
       - dpctl >=0.5.0a0
       - mkl-devel >=2021.1.1
+      - intel-openmp
+      - tbb
```
Comment (shssf) [inline conda-recipe/meta.yaml:17]: @Alexander-Makaryev  please confirm this but I think the project doesn't depend on TBB and OpenMP

**Comment by samir-nasibli** [comment]
@shssf @Alexander-Makaryev do we have some decision about `gold/2021` branch?

File: conda-recipe/meta.yaml (hunk 1 above)
Comment (samir-nasibli) [reply conda-recipe/meta.yaml:17]: Do we really need this pkgs? Properly check is needed.

Comment (Alexander-Makaryev) [reply conda-recipe/meta.yaml:17]: This is an attempt to solve internal build issue, dependencies are from MKL not DPNP. Issue is not a critical/stopper/etc. So I think we should remove them from here and find another way.

Comment (Vyacheslav-Smirnov) [reply conda-recipe/meta.yaml:17]: Ok, I will remove this packages

**Comment by Vyacheslav-Smirnov** [comment]
Wheel tests are passed, going to merge this PR

**Comment by Vyacheslav-Smirnov** [comment]
@shssf , are you OK with the changes?

**Comment by shssf** [comment]
I see lot of tests are failed.

**Comment by samir-nasibli** [comment]
This PR was not approved to merge.

**Pull request merged**